packages/api/src/moderation/subjects/user-list.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  ModerationSubjectUserList,
+  ModerationOpts,
+  ModerationDecision,
+} from '../types'
+
+export function decideUserList(
+  subject: ModerationSubjectUserList,
+  opts: ModerationOpts,
+): ModerationDecision {
+  // TODO handle labels applied on the list itself
```

Comment:
Just confirming this is intended to be left as a TODO for now 👍

Reply:
yeah pretty sure we're not returning labels on lists or feed gens yet

Reply:
yep